ElixrTalentConnect/SignUP/SignUpModelstructure.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  SignUpModelstructure.swift
+//  ElixrTalentConnect
+//
+//  Created by Devasurya on 16/01/24.
+//
+import UIKit
+import Foundation
+
+/// Data model for Signup view controller.
+enum DataModel {
```

Comment:
Please give more meaningful names to the types, if it for signup give something like `SignupDataModel`